packages/vue/examples/search-showcase/searchbox-inline-ai-response/.prettierrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+	tabWidth: 4,
+	printWidth: 100,
+	singleQuote: true,
```

Comment:
@SavvyShah Any reason to use a separate prettier config for this particular demo?